recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "nnpops" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/NNPOps/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0d770b1b6d48a6d76e6dce9b3da996da27a3767e6a0548bb5921d49c65adb635
+
+build:
+  skip: true  # [(not linux) or (cuda_compiler_version in (undefined, 'None'))]
+  number: 0
+  rpaths:
+    - lib/
+    - {{ SP_DIR }}/torch/lib
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64 or ppc64le]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64  2.17  # [linux64]
+    - make
+    - cmake
+  host:
+    - python 
+    - pytorch
+    - pytorch =*=*cuda*
+    - torchani
+    - mdtraj
+    - pytest
+  run:
+    - python
+    - pytorch
```

Comment:
```suggestion
    - {{ pin_compatible("pytorch", max_pin="x.x") }}
```

Reply:
Thanks! I saw something about using that in the build log, but the link 404'ed 
Also thanks again for your patience in walking me through this!